src/table/sticky-columns/use-sticky-columns.ts

```diff
@@ -260,9 +221,11 @@ interface UpdateCellStylesProps {
 }
 
 export default class StickyColumnsStore extends AsyncStore<StickyColumnsState> {
```

Comment:
Because I do not understand how this store works, I cannot review this file properly.

Count this approval as "I read it" instead of "I understand it"